realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -134,6 +133,8 @@
 
     public static final String DEFAULT_REALM_NAME = RealmConfiguration.DEFAULT_REALM_NAME;
 
+    private static final Object monitorForDefaultConfiguration = new Object();
```

Comment:
It's actually a monitor https://en.wikipedia.org/wiki/Monitor_(synchronization) , but yes we should use the single word for the same things.
I'll fix this.